### DISCUSSION TITLE
wasm runtime for pyiodide/pyscript (wish)

### DISCUSSION THREAD

**Discussion by giraypultar** [dashingsoft/pyarmor#1424] [Ideas]
We are trying to have our module being used from pyscript. https://pyscript.net/ which uses pyiodide.

Our module works fine as a pure wheel, but does not work when used with pyarmor.

I understand that pyarmor needs a wasm runtime to work inside pyscript/pyiodide.

This should be useful for anyone trying to get their scripts running with wasm.  There will probably be more people writing python for the browser, or other wasm enabled python environments such as wasm cloud functions

**Comment by jondy** [comment]
At this time, pyarmor doesn't work for WASM.
